### ISSUE TITLE
Flickering on RN 0.64

### ISSUE THREAD

**Issue by SergiOnGit** [archriss/react-native-snap-carousel#834]
Just test it on RN 0.64. enableSnap={true} not working good anymore. Test and you will understand.

**Comment by papidb** [comment]
I don't think this library is maintained anymore. Please I would like if anyone can point me to a library that does the stack animation provided by this library. Thank you!

**Comment by jfbn** [comment]
I am only experiencing this issue on Android.

**Comment by jfbn** [comment]
> I don't think this library is maintained anymore. Please I would like if anyone can point me to a library that does the stack animation provided by this library. Thank you!

What makes you think this? I see that the last update was February 11.

**Comment by Saad-Bashar** [comment]
Yes, getting this same issue on android. The snapping is not working.

**Comment by Saad-Bashar** [comment]
@jfbn, @SergiOnGit , did you figure out the issue, or still experiencing the same? Please let me know.

**Comment by jfbn** [comment]
> @jfbn, @SergiOnGit , did you figure out the issue, or still experiencing the same? Please let me know.

Still having the issue.

**Comment by SergiOnGit** [comment]
@Saad-Bashar It's not fixed yet. I switched to v4.0.0-beta.6 it works much better.

**Comment by SergiOnGit** [comment]
But the thing that plugin author isn't appearing makes me think that development is over.

**Comment by Saad-Bashar** [comment]
> @Saad-Bashar It's not fixed yet. I switched to v4.0.0-beta.6 it works much better.

By better do you mean that the snapping issue is fixed totally?

**Comment by SergiOnGit** [comment]
@Saad-Bashar Yes it is fixed. Just try.

**Comment by huang-ju** [comment]
I switched to v4.0.0 it works much better.

**Comment by ss2012** [comment]
removeClippedSubviews={false}可以解决这个问题，通过这个swiper库https://github.com/leecade/react-native-swiper/issues/1232

同样的问题产生的灵感，Carousel有同样的属性，试验过可以解决

**Comment by lxp-git** [comment]
Try undo this commit: https://github.com/facebook/react-native/commit/10314fe621e1649654e83df197adf657e0ca8363

**Comment by SergiOnGit** [comment]
@ss2012 Checked and it hasn't changed anything.

**Comment by appli-intramuros** [comment]
Try to activate momentum by adding enableMomentum to the component. (worked for me).

**Comment by SergiOnGit** [comment]
@appli-intramuros enableMomentum makes a bad animation, it is something like a slow motion. I better stay at version 4.

**Comment by dohooo** [comment]
Sorry, please allow me to advertise for my open source library! ~
I think this library [react-native-reanimated-carousel](https://github.com/zwh1666258377/react-native-reanimated-carousel) will solve your problem. It is a high performance and very simple component, complete with React-Native reanimated 2

**Comment by SergiOnGit** [comment]
@zwh1666258377 I'll try your module. It looks good and you shouldn't stop working on it.

**Comment by therealjmj** [comment]
v4.0.0 has a small issue, which is a 300-400ms delay on the `onSnapToItem` callback. 
however, it beats the flicker on Android. 

I used a stable version for iOS in my React Native app, and the beta for Android, in package.json like so:

```
...
"react-native-snap-carousel-beta": "npm:react-native-snap-carousel@^4.0.0-beta.6",
"react-native-snap-carousel-stable": "npm:react-native-snap-carousel@^3.9.1",
...
```

Then you'll have to require each version to include in jsx (warning: they're not type-safe).

```
const CarouselStable = require('react-native-snap-carousel-stable').default;
const CarouselBeta = require('react-native-snap-carousel-beta').default;
```

Hope this helps someone else achieve the proper UX!
